makefile.shade

```diff
@@ -8,4 +8,4 @@ k-standard-goals
 custom-goals
 
 #initialize if='Directory.Exists("src")'
-  exec program='dotnet' commandline='run -p ../../tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs' workingdir='${kestrelSrcDir}'
```

Comment:
Never mind... I just opened this in VS and notice that only the generated classes moved and not the GeneratedCode project itself.